library/tabs/src/TabsWrapper.php

```diff
@@ -68,6 +71,16 @@ public function genCss()
 }
 EOD;
         }
+
```

Comment:
The following should be within the above "if (!defined('INCLUDED_TW_ONETIME_CSS'))" conditional block, so it is not repeated when there are multiple tab sets.

Reply:
Done.